### PR TITLE
Update short description and logo

### PR DESCRIPTION
**Pull request by Julievrz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Update short description and logo.
| Type?         | improvement 
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/15858
| How to test?  | Please check that short description and logo are updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## About
 
-Displays a block with links relative to a user's account.
+Enrich your stats, add a registration progress tab to analyze your visitors’ behavior.
```
Comment (NeOMakinG) [inline README.md:5]:
```suggestion
Enrich your stats, add a registration progress tab to analyze your visitor’s behavior.
```

Isn't it like this?

File: ps_customeraccountlinks.php [hunk 2]
```diff
@@ -43,7 +43,7 @@ public function __construct()
         parent::__construct();
 
         $this->displayName = $this->trans('My Account block', [], 'Modules.Customeraccountlinks.Admin');
-        $this->description = $this->trans('Displays a block with links relative to a user\'s account.', [], 'Modules.Customeraccountlinks.Admin');
+        $this->description = $this->trans('Enrich your stats, add a registration progress tab to analyze your visitors’ behavior.', [], 'Modules.Customeraccountlinks.Admin');
```
Comment (NeOMakinG) [inline ps_customeraccountlinks.php:46]:
```suggestion
        $this->description = $this->trans('Enrich your stats, add a registration progress tab to analyze your visitor’s behavior.', [], 'Modules.Customeraccountlinks.Admin');
```

same question

File: README.md (hunk 1 above)
Comment (Julievrz) [reply README.md:5]: @NeOMakinG I think we want it to be plural here so it is visitors' ☺️

**Review by NeOMakinG — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by atomiix** [comment]
Thanks @Julievrz

**Pull request merged**